### PR TITLE
Add recap blog post for SQL training 2

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: _posts/2023-11-01-sql-training-2.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+layout: post
+title: "Advanced analytics with SQL and Trino"
+author: "Manfred Moser"
+excerpt_separator: <!--more-->
+image: /assets/trino-sql.png
+show_pagenav: false
+---
+
+In the second part of our training series [Learning SQL with Trino from the
+experts]({% post_url 2023-09-27-training-series %}) Martin Traverso and myself
```
Comment (colebow) [inline _posts/2023-11-01-sql-training-2.md:11]:
```suggestion
experts]({% post_url 2023-09-27-training-series %}) Martin Traverso and I
```

File: _posts/2023-11-01-sql-training-2.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+built on top of the foundational knowledge from the [first training session]({%
+post_url 2023-10-18-sql-training-1 %}). We continued to learn more about data
+types and working with them, including the important strings, numeric, temporal
```
Comment (colebow) [inline _posts/2023-11-01-sql-training-2.md:14]:
```suggestion
types and working with them, including the important strings, numeric, temporal,
```

File: _posts/2023-11-01-sql-training-2.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+The recording of the event allows you to watch it all as if you attended live,
+jump to specific sections as desired, or pause while you follow along with the
+demos:
+
+{% youtube S-mfueDmXds %}
+
+Following are a couple of specific timestamps for interesting
+topics snippets:
+
+* [Temporal data type](https://www.youtube.com/watch?v=S-mfueDmXds&t=601s)
```
Comment (colebow) [inline _posts/2023-11-01-sql-training-2.md:28]:
```suggestion
* [Temporal data types](https://www.youtube.com/watch?v=S-mfueDmXds&t=601s)
```

File: _posts/2023-11-01-sql-training-2.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+* [Strings](https://www.youtube.com/watch?v=S-mfueDmXds&t=1920s)
+* [Numeric](https://www.youtube.com/watch?v=S-mfueDmXds&t=2442s)
```
Comment (colebow) [inline _posts/2023-11-01-sql-training-2.md:30]:
```suggestion
* [Numeric types](https://www.youtube.com/watch?v=S-mfueDmXds&t=2442s)
```

File: _posts/2023-11-01-sql-training-2.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+* [General overview slide deck for the
+  series]({{site.baserurl}}/assets/blog/sql-training-series-starburst-2023.pdf),
+  with links to resources like our [community
+  chat]({{site.baserurl}}/slack.html)
+* Slides deck for [Advanced analytics with SQL and
+  Trino](https://trinodb.github.io/presentations/presentations/sql-adv-analytics/index.html),
+  including a file with all SQL statements ready to go
+* [Trino: The Definitive Guide]({{site.baserurl}}/trino-the-definitive-guide.html)
+
+We are half way through the series, and there is lots more to cover. Don't
```
Comment (colebow) [inline _posts/2023-11-01-sql-training-2.md:48]:
```suggestion
We are halfway through the series, and there is lots more to cover. Don't
```

File: _posts/2023-11-01-sql-training-2.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+forget to register for the next ones, join us to ask specific questions, and
```
Comment (colebow) [inline _posts/2023-11-01-sql-training-2.md:49]:
```suggestion
forget to register for the next sessions, join us to ask specific questions, and
```

**Review by colebow — APPROVED** [review]
Some grammar nits, but LGTM.

**Review by martint — APPROVED** [review]

**Comment by mosabua** [comment]
Thanks @martint

**Pull request merged**